app/views/idv/in_person/ready_to_verify/show.html.erb

```diff
@@ -9,78 +9,36 @@
       ) %>
 <% end %>
 
-<%= image_tag(
-      asset_url('idv/user-in-person.svg'),
-      width: 136,
-      height: 136,
-      class: 'display-block margin-x-auto margin-bottom-4',
-      alt: '',
-    ) %>
-
 <%= render PageHeadingComponent.new(class: 'text-center') do %>
-  <%= t('in_person_proofing.headings.barcode', app_name: APP_NAME) %>
-<% end %>
-
-<%= render AlertComponent.new(class: 'margin-y-4', text_tag: :div) do %>
-  <p class="margin-bottom-1"><strong><%= t('in_person_proofing.body.barcode.deadline', deadline: @presenter.formatted_due_date) %></strong></p>
-  <p class="margin-bottom-0"><%= t('in_person_proofing.body.barcode.deadline_restart') %></p>
+  <%= t('in_person_proofing.headings.barcode') %>
```

Comment:
Nit: Could we force a line break (or something else) to keep "state-issued ID" together on one line?

Reply:
if we convert the space to the unicode equivalent of `&nbsp;` that would keep "issued" and "ID" together, and then turn the hyphen into a [non-breaking hyphen](https://stackoverflow.com/a/8753317) we could do that

Reply:
Yeah, would prefer using a non-breaking character like @zachmargolis suggested, since forced line breaks tend to have unexpected consequences for different screen sizes (mobile, etc.).

Reply:
Further conversation/decisioning [happened here](https://gsa-tts.slack.com/archives/C03N6KQBAKS/p1682533879701429).

Reply:
For documentation - this was addressed and this is how it looks after implementing the changes:
![image](https://user-images.githubusercontent.com/13384921/235261496-44061573-bad4-4961-b5f8-11f324e64fe1.png)